samples/highcharts/accessibility/accessible-dynamic-data-animating/demo.css

```diff
@@ -33,3 +37,23 @@ th {
     font-weight: 600;
     padding: 10px;
 }
+
+.highcharts-demo-button {
```

Comment:
It might be outside of the review scope, but it would be nice to have a slider instead of a button when it is used to toggle on-off something. Like Dashboards' edit mode: 
![image](https://github.com/user-attachments/assets/8b45e5c0-7923-47fe-be2d-05381bcdd63e)